build_tools/buildkite/cmake/android/arm64-v8a/benchmark2.yml

```diff
@@ -10,6 +10,7 @@ steps:
   - label: "Build"
```

Comment:
unrelated, but isn't it about time to rename this `benchmark.yml` instead of `benchmark2.yml`? :P